### PR TITLE
Marines slides + katas

### PR DESCRIPTION
**Pull request by 88mary256**

### CODE REVIEW

File: src/marines/numbers.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        return "not supported"
+
+
+def sum_to(n):
+    sum = 0
+    for i in range(1, n+1):
+        sum += i
+    return sum
+
+print "---- IS ODD OR EVEN ----"
```
Comment (sperBruno) [inline src/marines/numbers.py]: you must apply unit test

File: src/marines/my_console.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    for index in range(0, int(size)):
+        number = raw_input("please set the " + str(index + 1) + " value of the array: ")
+        myArray.append(number)
+    return myArray
+
+
+def function2(myArray):
+    print myArray
+
+def function3():
```
Comment (sperBruno) [inline src/marines/my_console.py]: what does function3 mean?

File: src/marines/my_console.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+def function1():
```
Comment (sperBruno) [inline src/marines/my_console.py]: what does function mean?

File: src/marines/my_string.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
+
+
+def get_all_urls(line, start, urls):
+    size = len(line)
+    index = line.find("http://", start, size)
+    endIndex = line.find(" ",index,size)
+    endIndex = size if (endIndex == -1) else endIndex
+    urls.append(line[index:endIndex])
+    if ( endIndex == size):
```
Comment (sperBruno) [inline src/marines/my_string.py]: apply ternary condition?

File: src/marines/my_string.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    return s.replace(old, new);
+
+print replace("hi every bohidy whihats wronghi", "hi", "")
+
+def countLetters(line):
+    letters = {}
+    for i in range(len(line)):
+        char = line[i].lower()
+        if (char.isalpha()):
+            if (char in letters):
```
Comment (sperBruno) [inline src/marines/my_string.py]: apply ternary condition?

File: src/marines/numbers.py [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+def isPrime(number):
+    for i in range(2, number):
+        if(number % i == 0):
+            return False
+    return True
+
+def primeList(init,end):
+    primes = []
+    for number in range(init,end +1):
```
Comment (sperBruno) [inline src/marines/numbers.py]: apply correct python format

File: src/marines/operators.py [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    result = perform_operation(op, a, b)
+    print "{} {} {} = {}".format(a, op, b, result)
+    return result
+
+
+c = perform_operation("*", "5", "6")
+print c
+perform_and_print_operation("*", "5", "6")
+perform_and_print_operation("+", "2", "5")
+perform_and_print_operation("-", "2", "5")
```
Comment (sperBruno) [inline src/marines/operators.py]: apply unit test

File: src/marines/time.py [hunk 8]
```diff
@@ -0,0 +1,10 @@
+months = {"January":31, "February":28, "March": 31, "April":30, "May":31, "June":30, "July":31, "August":31,
+          "September":30, "October":31, "November":30, "December":31}
+
+def days_in_month(month):
+    return months[month]
```
Comment (sperBruno) [inline src/marines/time.py]: apply unit tests

File: src/marines/numbers.py (hunk 1 above)
Comment (88mary256) [reply src/marines/numbers.py]: done

File: src/marines/my_console.py (hunk 2 above)
Comment (88mary256) [reply src/marines/my_console.py]: it is for call function1 and function2.
Renamed by load_ and_print_array

File: src/marines/my_console.py (hunk 3 above)
Comment (88mary256) [reply src/marines/my_console.py]: I use the name provided in the slide.
Renamed by load_integer_array

**Pull request merged**